iterative_verifier.go

```diff
@@ -68,22 +68,23 @@ func (r *ReverifyStore) Add(entry ReverifyEntry) {
 	}
 }
 
-func (r *ReverifyStore) FlushAndBatchByTable(batchsize int) []ReverifyBatch {
+func (r *ReverifyStore) FlushAndBatchByTable(c *CursorConfig) []ReverifyBatch {
```

Comment:
Hmm maybe we shouldn't change this one? Since we don't really have a problem with this.

Reply:
Oh this is the interactive verifier. We should leave it be as that component is deprecated.

Reply:
ok, so for this we won't look in the override map and just use the BatchSize default